CMakeLists-legacy.txt

```diff
@@ -0,0 +1,661 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
We can look at the commit history if we need to, so I think we'll be fine without this file.

Reply:
On Windows, it should be

SQLite
```
-DCMAKE_PREFIX_PATH="C:/dev/OpenSSL-Win64;C:/dev/SQLite-Win64;C:/dev/Qt/5.12.12/msvc2017_64"
```

SQLCipher
```
-DCMAKE_PREFIX_PATH="C:/dev/OpenSSL-Win64;C:/git_repos/SQLCipher-Win64;C:/dev/Qt/5.12.12/msvc2017_64"
```

Those are paths set in the old configuration.
And same for 32bit